brands/shop/clothes.json

```diff
@@ -294,9 +294,7 @@
     "tags": {
       "brand": "Cache Cache",
       "name": "Cache Cache",
-      "operator": "Groupe Beaumanoir",
-      "operator:wikidata": "Q3117333",
-      "operator:wikipedia": "fr:Groupe Beaumanoir",
+      "operator:wikidata": "Q63064086",
```

Comment:
Shouldn't this be `brand:wikidata`?

Why remove all the operator tagging?

Reply:
Yeah, Q6306408 should be the brand:wikidata entry and Q3117333 should stay as the operator. It would also be good if you filled out the description and name for Q63064086. Otherwise, there's no way for anyone to find it. Plus, it helps in the brands list here if they have those details http://osmlab.github.io/name-suggestion-index/brands/index.html

Reply:
> Shouldn't this be `brand:wikidata`?
> 
Yes indeed, this is a typo.

> Why remove all the operator tagging?
Cache Cache can be franchise (https://www.toute-la-franchise.com/franchise-15291-cache-cache.html 🇫🇷) so I found ackward all future Cache Cache POI will be declared as operated by Groupe Beaumanoir